docs/actors/development/source_code.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Source code
-description: Documentation of Apify actors - storing your actor's source code.
+description: Define the location where your actor's code is stored. Build your actor from a GitHub repo, upload a Zip file or write your actor in the in-app editor.
```

Comment:
This is not only about the location but generally about source code - for example input schema format.